### ISSUE TITLE
Investigate: non-trivial number of users have no app ID in Amplitude

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#2556]
### Vision statement / What / Requirements
I noticed a non-trivial numbers have no app ID (e.g. org.mozilla.tv.firefox) in Amplitude: https://analytics.amplitude.com/mozilla-corp/chart/hosooc4

I wonder if this could be indicative of problems in our data, our app, etc.

### Impact
Unknown. Some speculations on the cause (from which you can speculate on impact):
- A data conversion error in Amplitude
- We scrub data before it enters Amplitude
- The app is reporting this value before it's available to the app (is this possible?)
- Someone forked the app and removed the app ID, but we're still seeing their telemetry
- There's a bug in disabling data so we still transmit, just without an app ID

Could this be related to the Fretboard spike? #2469

**Comment by athomasmoz** [comment]
@mcomella what's the app id in Amplitude used for?

**Comment by mcomella** [comment]
> @mcomella what's the app id in Amplitude used for?

The app ID is the application's canon identifier: the Google Play Store and Android only allows one app with a given identifier (I assume Amazon is the same). Ours is `org.mozilla.tv.firefox`.

I noticed the app ID was introduced in maybe 3.1.3 or 3.2: the code was added 2018-12-19. I split up users without an app_id by release version ([results](https://analytics.amplitude.com/mozilla-corp/chart/ah9tn0r)) and it turns out all the users with "none" app IDs are on versions prior to this. In fact, most of these users are on 2.0.1, where many of our users are stuck because they need to accept a permissions upgrade.

I now think having no app ID is to be expected for these users and no longer concerning: closing this issue as WORKSFORME.
